R/installspec.r

```diff
@@ -14,11 +20,11 @@ installspec <- function(user = TRUE) {
             break
         }
     }
-    
+
     if (!found_binary)
-        stop('Jupyter or IPython 3.0 has to be installed but could neither run “jupyter” nor “ipython”, “ipython2” or “ipython3”.
-             (Note that “ipython2” is just IPython for Python 2, but still may be IPython 3.0)')
-    
+        stop('Jupyter or IPython 3.0 has to be installed but could neither run "jupyter" nor "ipython", "ipython2" or "ipython3".
+             (Note that "ipython2" is just IPython for Python 2, but still may be IPython 3.0)')
```

Comment:
ok, I'm passionate about the smart code thing (or using any non-unicode char) in code . Or better: violently against it: it's messing with windows cmd output (such as git diff) and it's breaking when I c&p it into an example call. Smart quotes should be outlawed in any source code files and even in published documentation...

R seems to agree, at least it issues a warning during my builds...

Anyway: I will revert these parts as the have nothing to do with the original issue...